Feature/option clear jvmf

AdozenBABYdogs opened the pull request:
**When merged this pull request will:**
- Describe what this pull request will do
- Each change in a separate line
- Include documentation if applicable

This PR adds the ability to remove JVMF default messages via the addon settings

AdozenBABYdogs committed:
Added in option to remove default JVMF messages
JVMF messages can now be removed via clicking the check box in the
settings menu of the UH-60M.

Cannot build .pbos on linux, need to find a solution to that

AdozenBABYdogs committed:
Forgot to add one line of code
The one that actually adds the setting

jonpas commented on addons/uh60_jvmf/XEH_preStart.sqf:
```diff
@@ -1,3 +1,11 @@
+	,"CHECKBOX"
+	,["Enable JVMF Default Messages", "The JVMF Default messages provide an example of usage of the JVMF"]
+	,["UH-60M", "JVMF"]
+	[true];
```
This doesn't look correct.

AdozenBABYdogs replied:
Yep, I see it.  I'm missing a comma.  Does the syntax for cba_fnc_addSetting look correct?

AdozenBABYdogs replied:
I also think the order on where I'm executing the code if off as well, and this function belongs in the "XEH_preInit.sqf"

jonpas replied:
Bool, not an array.

AdozenBABYdogs replied:
Thank you

AdozenBABYdogs committed:
rearanged items order
hopefully this fixes things

AdozenBABYdogs committed:
added missing comma

AdozenBABYdogs committed:
fixed array -> bools

AdozenBABYdogs committed:
fixed typo: Misplaced semicollin

AdozenBABYdogs committed:
resolving bug fixfor JVMF messages not displaying
https://github.com/Project-Hatchet/H-60/issues/354

AdozenBABYdogs committed:
Additonal Bug fixes for JVMF 0 messages

AdozenBABYdogs committed:
attemping to fix JVMF not displaying

AdozenBABYdogs committed:
re-adjusted jvmf to try anddisplay message

AdozenBABYdogs commented:
JVMF requires rewrite in order to support 0 messages in the table at the time of use

AdozenBABYdogs committed:
resolving bug fixfor JVMF messages not displaying

https://github.com/Project-Hatchet/H-60/issues/354

# Conflicts:
#	addons/uh60_jvmf/functions/fnc_drawJVMF.sqf
Fixing JVMF from an earlier stage

AdozenBABYdogs committed:
Added in JVMF delete buttonTODO:  fix new bugs

AdozenBABYdogs committed:
JVMF Delete message works as intended

added in functionality to delete the messages
hides JVMF UI when all messages are deleted

TODO/Forseen issues:
CTAB messages will get re-added into JVMF if they are deleted

AdozenBABYdogs commented:
Ready for merge

JVMF rewrite completed, JVMF hides when no messages are present.  JVMF Delete message button added to remove erroneous/non applicable JVMF messages.

Riverman02 reviewed (approved):
Let's try it

BroBeansCPG committed:
layout shift & debug messages
